src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -1328,3 +1328,51 @@ def create_lvs(volume_group, parts=None, size=None, name_prefix='ceph-lv'):
             create_lv(name_prefix, uuid.uuid4(), vg=volume_group.name, extents=extents, tags=tags)
         )
     return lvs
+
+
+#############################################################
+#
+# New methods to get PVs, LVs, and VGs.
+# Later, these can be easily merged with get_api_* methods
+#
+###########################################################
+
+PV_FIELDS = 'pv_name,pv_tags,pv_uuid,vg_name,lv_uuid'
+VG_FIELDS = 'vg_name,pv_count,lv_count,snap_count,vg_attr,vg_size,vg_free,vg_free_count'
+LV_FIELDS = 'lv_tags,lv_path,lv_name,vg_name,lv_uuid,lv_size'
+
+def get_pvs(fields=PV_FIELDS, sep='";"', filters=None):
+    args = ['pvs', '--no-heading', '--readonly', '--separator=' + sep,
+            '-o', fields]
+    if filters:
+        args.extend(['-S', filters])
+
+    stdout, stderr, returncode = process.call(args, verbose_on_failure=False)
+
+    if stdout in [[], '']:
+        return None
```

Comment:
I think this is superflous. `_output_parser` should return an empty list when stdout is empty. Best return an empty list instead of `None`.